src/BizHawk.Emulation.Cores/Computers/DOS/DOSBox.cs

```diff
@@ -35,11 +35,13 @@ public partial class DOSBox : WaterboxCore
 		};
 
 		private LibDOSBox _libDOSBox;
-		private readonly List<IRomAsset> _floppyDiskAssets;
+		private readonly List<IRomAsset> _romAssets;
 		private readonly List<IDiscAsset> _discAssets;
 
 		// Drive management variables
 		private List<IRomAsset> _floppyDiskImageFiles = new List<IRomAsset>();
+		private IRomAsset _hardDiskImageFile = null;
+		private ulong _hardDiskImageFileSize = 0;

```

Comment:
This should probably be `int` or `uint`, because putting anything larger in here will not work.